File: kattis/bog/chapter1/real_life_harder/bungeejump/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	double k, l, s, w;
	while (scanf("%lf%lf%lf%lf", &k, &l, &s, &w), k != 0 || l != 0 || s != 0 || w != 0) {
		double dl = max(s-l, 0.0);
		double vs = 9.81*s*2 - k*dl*dl/w;
		if (vs < 0)
			printf("Stuck in the air.\n");
		else if (sqrt(vs) <= 10)
			printf("James Bond survives.\n");
		else
			printf("Killed by the impact.\n");
	}
}

File: kattis/bog/chapter1/real_life_harder/bungeejump/main.py
import math
from decimal import Decimal
g = Decimal("9.81")
while True:
	k, l, s, w = map(Decimal, input().split())

	if not (k or l or s or w):
		break
	vs = g*s*2 - (s-l)*(s-l)*k / w
	if vs < 0:
		print("Stuck in the air.")
	elif math.sqrt(vs) <= 10:
		print("James Bond survives.")
	else:
		print("Killed by the impact.")
